joomla/joomla-cms
# TinyMCE drag and drop bug fixes

Pull Request for Issue ##14906 .

### Summary of Changes
Apart from fixing the bug in #14906 this PR also fixes another bug with the toolbar tooltip rendered in wrong position after the first image upload.
Also moves the plugin javascript code out of the tinyMCE/plugins directory (will make updates easier)


### Testing Instructions
Edit the configuration and provide a custom path for the drag and drop upload, this setting is per tab option so make sure that you are using the right one!!!
Try to upload an image,
Check where the image is stored and also try to see if the tooltips in bold, italics etc is in the right position

Apply the patch and repeat


### Expected result
Use the provided path


### Actual result
Is not using the provided path


### Documentation Changes Required
None, bug fix
